scripts/build_powerplants.py

```diff
@@ -247,9 +247,12 @@ def add_custom_powerplants(ppl, inputs, config):
     # if isinstance(custom_ppl_query, str):
     #     add_ppls.query(custom_ppl_query, inplace=True)
```

Comment:
I know pypsa-eur is using line 247 and 248. Can you figure out its purpose?
I think your merge and replace option could make lines 247 and 248 not necessary. Maybe we can remove the comment?

Reply:
I have checked the code and it seems that it is left to enable some query to custom powerplants being added. We can leave them as commented or we can remove them.

Reply:
But it is better to remove them, since if enabled it might contradict with the proposed PR.